core/src/test/scala/org/apache/spark/rdd/AsyncRDDActionsSuite.scala

```diff
@@ -197,4 +197,50 @@ class AsyncRDDActionsSuite extends SparkFunSuite with BeforeAndAfterAll with Tim
       Await.result(f, Duration(20, "milliseconds"))
     }
   }
+
+  private def testAsyncAction[R](action: RDD[Int] => FutureAction[R])
+    (starter: => Semaphore) : Unit = {
```

Comment:
To clarify: If I put the parameters in the same parameter list, the compiler forces me to be a lot more verbose at the call sites (with curly braces, `=>`, and so forth)